rasa/core/tracker_store.py

```diff
@@ -63,9 +67,16 @@
 # default value for key prefix in RedisTrackerStore
 DEFAULT_REDIS_TRACKER_STORE_KEY_PREFIX = "tracker:"
 
+CONNECTION_ERRORS = (
+    NoCredentialsError,
+    psycopg2.OperationalError,
```

Comment:
is there a generic SQLAlchemy error which covers this one as well (e.g. sqlachemy.exc.OperationalError). I'm brining this up as this wouldn't work for other types of tracker stores or other SQL dialects (e.g. oracle)

Reply:
actually, I think we can remove this, as SQLAlchemy wraps exceptions and re-raise using their own class `sqlalchemy.exc.OperationalError`. Good catch